Hero title: Add right aligned styling

swetabar opened the pull request:
<!--- Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after): -->

Fix #141 

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/caesars-palace/drafts/swbarman/hero-align-right
- After: https://issue-141-hero-title--caesars--hlxsites.hlx.page/caesars-palace/drafts/swbarman/hero-align-right

## 📝 Description:
  
![Screen Shot 2023-04-04 at 8 47 53 AM](https://user-images.githubusercontent.com/15715290/229796340-62b7c403-a7bc-427e-a20b-87b5b563b6a2.png)
![Screen Shot 2023-04-04 at 8 48 13 AM](https://user-images.githubusercontent.com/15715290/229796443-ddfbadbd-a6ff-4a30-a2a4-bfac9f623f45.png)

tmathern commented:
The `before` link returns 404 for me. https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/hero could be used for comparison too.

tmathern reviewed (commented):
Can you also check this works for heroes with video and carousel? You can use a page like https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/hero which gathers different kinds of hero headers.

swetabar commented:
> Can you also check this works for heroes with video and carousel? You can use a page like https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/hero which gathers different kinds of hero headers.

I've added carousel and video to the word doc. 
Added some modifications to the CSS for alignment for carousels and videos.
